map/src/context/AppContext.js

```diff
@@ -354,6 +354,7 @@ export const AppContextProvider = (props) => {
         }
     });
 
+    const [localClientsTracks, setLocalClientsTracks] = useState(localStorage.getItem('localClientsTracks') !== null ? JSON.parse(localStorage.getItem('localClientsTracks')) : []);
```

Comment:
Util or Manager or LocalTracksStorage? (JSON.parse not here)

Reply:
```
LocalTracksStorage.loadTracks {
localStorage.getItem('localClientsTracks') !== null ? JSON.parse(localStorage.getItem('localClientsTracks')) : []
}
```

Reply:
`+`